data/Warm_Cold.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+number = random.randint(1, 50)
+value = input("Please enter a number:\n")
+counter = 0
+while True:
+    counter += 1
+    if number == int(value):
+        print('you won')
```

Comment:
Komentarze dla użytkowników powinny być jak najładniejsze z zachowaniem interpunkcji gramtyki itd.

Reply:
zmienione na 'You won, congratulations!'